deploymentTemplates/aksarc-ARM/HCI/azuredeploy.parameters.json

```diff
@@ -33,36 +39,46 @@
             "value": 0
         },
         "kubernetesVersion": {
-            "value": ""
+            "value": "" // TODO: Fill in parameter value
         },
-        "provisionedClustersApiVersion":{
-            "value": "2024-01-01"
-        },
-		"connectedClustersApiVersion" : {
-			"value": "2024-07-15-preview"
-		},
         "controlPlaneNodeCount": {
             "value": 1
         },
         "controlPlaneIp": {
-            "value": "100.72.19.162"
+            "value": "" // TODO: Fill in parameter value
         },
         "controlPlaneVMSize": {
             "value": "Standard_A4_v2"
         },
-        "vnetSubnetIds": {
-            "value": [
-                "TODO: Enter the ARM ID of the vnet subnet"
-            ]
-        },
         "podCidr": {
             "value": "10.244.0.0/16"
         },
         "networkPolicy": {
             "value": "calico"
         },
         "customLocation": {
-            "value": "TODO: Enter the custom location ARM ID" 
+            "value": "" // TODO: Fill in parameter value
```

Comment:
There are no comment notations like this in JSON. Same comment for everywhere else that uses `//`.